src/components/solution-control/solution-create-dialog/solution-create-dialog.tsx

```diff
@@ -21,23 +22,37 @@ export const SolutionCreateDialog = ({
   onCreate,
 }: ISolutionCreateDialogProps): ReactElement => {
   const [solutionName, setSolutionName] = useState('')
+  const [isValidationError, setIsValidationError] = useState(false)
 
-  const error = useMemo(
+  const isSameNameError = useMemo(
     () => solutions?.some(solution => solution.name === solutionName),
     [solutionName, solutions],
   )
 
+  const hasError = isValidationError || isSameNameError
+  const errorText = isValidationError
+    ? t('filter.notValidName')
+    : t('solutionControl.createDialog.solutionNameAlreadyExists', {
+        controlName,
+        solutionName,
+      })
+
   useEffect(() => {
     !isOpen && setSolutionName('')
   }, [isOpen])
 
+  useEffect(() => {
+    const isValid = !solutionName.length || validatePresetName(solutionName)
```

Comment:
and here:
`setIsSameName(!!solutions?.some(solution => solution.name === solutionName))`